LDM-148.tex

```diff
@@ -396,6 +400,24 @@ \section{Prompt Base Enclave}\label{prompt-base-enclave}
 The current design exposes each service as a separate entity with potentially different service levels.
 A DMCS component still exists, but it is an implementation feature of the services' design rather than an architectural component.
 
+The overall service level for this enclave should be high, with the goal being to have the enclave functional whenever the Observatory is taking data.
+
+Of the services within the enclave, the Header Generator component of the Archiving service is critical to observing and should have a target uptime equal to the Engineering and Facility Database.
+
+The Observatory Operations Data, OCS Driven Batch, Prompt Processing Ingest, and Telemetry Gateway services are important for maintaining high-quality observing; lack of these services constitutes a degraded mode of operation without useful feedback loops.
```

Comment:
OCS Driven Batch: even when configured to execute in the Offline Production Enclave at NCSA?

Reply:
Added a sentence to indicate that reconfiguration to run on CC might be necessary.